src/Sylius/Behat/Context/Api/Admin/ManagingTaxonImagesContext.php

```diff
@@ -88,7 +95,14 @@ public function iRemoveTheFirstImage(): void
         $taxonImage = $taxon->getImages()->first();
         Assert::notFalse($taxonImage);
 
-        $this->client->delete(Resources::TAXON_IMAGES, (string) $taxonImage->getId());
+        $builder = RequestBuilder::create(
+            sprintf('/api/v2/admin/taxons/%s/images/%s', $taxon->getCode(), $taxonImage->getId()),
+            Request::METHOD_DELETE,
+        );
+        $builder->withHeader('HTTP_Authorization', 'Bearer ' . $this->sharedStorage->get('token'));
+        $builder->withHeader('CONTENT_TYPE', 'application/ld+json');
+
+        $this->client->request($builder->build());
```

Comment:
As this part is duplicated in 2 methods, it could be extracted to the private one